Gemfile

```diff
@@ -40,6 +40,7 @@ group :development, :test do
   gem "pry-byebug"
   gem "rspec_junit_formatter"
   gem "rspec-rails", "~> 5.1"
+  gem "rswag"
```

Comment:
this needs to move to outside the dev/test section as we need it in production for UAT/STG/PROD deployments

Reply:
also the gem "apipie-rails", ">= 0.7.1" [line 31] can be removed from gemfile